src/core.js

```diff
@@ -425,6 +425,16 @@ jQuery.extend({
 	support: support
 });
 
+// jsHint would error on this code due to the Symbol not being defined in ES5.
```

Comment:
Maybe. But one of the ways we currently ensure we don't completely blow up in environments like Node.js is to restrict the list of defined globals to a very small list (mostly `window` & timer functions). So I preferred to not add another one.